Neos.Flow/Classes/Persistence/Doctrine/DataTypes/JsonArrayType.php

```diff
@@ -93,58 +68,54 @@ public function getMappedDatabaseTypes(AbstractPlatform $platform)
      * @param mixed $value The value to convert.
      * @param AbstractPlatform $platform The currently used database platform.
      * @return array The PHP representation of the value.
+     * @throws ConversionException
+     * @throws TypeConverterException
      */
-    public function convertToPHPValue($value, AbstractPlatform $platform)
+    public function convertToPHPValue($value, AbstractPlatform $platform): array
     {
-        $this->initializeDependencies();
+        $value = parent::convertToPHPValue($value, $platform);
 
-        switch ($platform->getName()) {
-            case 'postgresql':
-                $value = (is_resource($value)) ? stream_get_contents($value) : $value;
```

Comment:
for those who are wondering, this logic is already implemented in:
`\Doctrine\DBAL\Types\JsonArrayType::convertToPHPValue`